Index: test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test.py b/test.py
new file mode 100644
--- /dev/null	(date 1651239183498)
+++ b/test.py	(date 1651239183498)
@@ -0,0 +1,11 @@
+from tkinter import *
+
+top = Tk()
+
+top.geometry("200x200")
+
+spin = Spinbox(top, from_=1, to=9999)
+
+spin.pack()
+
+top.mainloop()
